libs/mutantschemata/cpp_source/rewrite.hpp

```diff
@@ -24,23 +24,171 @@
 #define MS_CLANG_REWRITE
 
 
-// global vars
+// defines to control what happens \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\|
+//#define onlyAddSub
+#define debugDuplicateMutantPrevention
+// config mutant schemata (not really implemented)
+bool negateBinaryOperators = true;
```

Comment:
I would recommend moving all globals to a "context" for the schemata. This enable such things as multithreading the schemata generation, easier reset (remember the case where an application isn't possible to inject all mutants in) etc.